Dockerfile

```diff
@@ -21,8 +21,12 @@
 #
 # (MIT License)
 
+# Upstream Build Args
+ARG OPENAPI_IMAGE=openapitools/openapi-generator-cli:v4.1.2
```

Comment:
Shouldn't this be pulled from algol60?

Reply:
The version of openapi generator in algol60 is not compatible with BOS spec.

Reply:
Then we should get a compatible version on algol60.

Reply:
That's fine, but not the point of this mod. https://connect.us.cray.com/jira/browse/CASMCMS-7697